Gems/Atom/Tools/ShaderManagementConsole/Code/Source/Window/ShaderManagementConsoleWindow.cpp

```diff
@@ -119,10 +120,57 @@ namespace ShaderManagementConsole
         return result.Native();
     }
 
+    void ShaderManagementConsoleWindow::ErrorMessageBoxesForDocumentVerification(const DocumentVerificationResult& verification)
```

Comment:
If the verify function prints out more elaborate messages for all of these conditions, you can capture them using the trace recorder class, which connects the log bus and records any traces, warnings, or errors printed within scope. Then you can have a single message box display a dump from the trace recorder. That should happen automatically when loading or saving the document.

https://github.com/o3de/o3de/blob/59065d9327841543f60887ef863d50dd9d8ccfe0/Gems/Atom/Tools/AtomToolsFramework/Code/Source/Document/AtomToolsDocumentSystem.cpp#L170

https://github.com/o3de/o3de/blob/59065d9327841543f60887ef863d50dd9d8ccfe0/Gems/Atom/Tools/AtomToolsFramework/Code/Source/Document/AtomToolsDocumentSystem.cpp#L197-L199